simplifierBrowse.rst

```diff
@@ -1,17 +1,24 @@
-Browse Simplifier
+Searching on Simplifier
 ========================
-Simplifier content is organized at four levels: Users, Resources, Projects and Organizations. Simplifier allows you to get in touch with other users as well as to browse organizations, projects and resources from the homepage. 
-
-Find other Simplifier users
----------------------------
-When you visit the ``Members`` tab on a `project page <simplifierProjects.html#project-page>`_, you see which Simplifier users are involved in the project. By clicking on a project member, you will navigate to his or her `user profile <simplifierPersonalContent.html#user-profile>`_. Here you can find more information about this user, such as his or her activity on Simplifier and other projects this user is involved in. You can also get in touch with this user by using his or her contact details or sending a direct message.
+Simplifier allows you to access FHIR data models from all around the world. You can search for Organizations and access their publicly available projects. It shows you how the best Health Organizations use FHIR and you can save time reinventing the wheel. 
```

Comment:
It shows you how the best Health Organizations use FHIR and HOW you can save time reinventing the wheel.